overviews/core/_posts/2012-09-20-futures.md

```diff
@@ -36,6 +36,134 @@ in the case of an application which requires blocking futures, for an underlying
 environment to resize itself if necessary to guarantee progress.
 -->
 
+A typical future look like this:
```

Comment:
That's not how a Future looks, I think it is important to distinguish between Future-the-object and Future.apply-the async-computation-resulting-in-a-Future-the-object.

I.e.

val f = Future.successful(())
vs
val f = Future(())